toolbox/gpu-operator/operand-build/operands/toolkit.json

```diff
@@ -0,0 +1,37 @@
+{
+  "build_operand_git_repo": "https://gitlab.com/nvidia/container-toolkit/container-config.git",
+  "build_operand_git_ref": "master",
+  "build_operand_namespace": "gpu-operator-resources",
+  "build_operand_imagestream_name": "container-toolkit",
+  "build_operand_tag": "latest",
+  "build_operand_dockerfile": "docker/Dockerfile.ubi8",
+  "build_operand_docker_context": ".",
+  "build_operand_docker_build_args": [
+    {
+      "name": "GOLANG_VERSION",
+      "value": "1.16.4"
+    },
+    {
+      "name": "VERSION",
+      "value": "1.5.0"
+    },
+    {
+      "name": "LIBNVIDIA_CONTAINER_VERSION",
+      "value": "1.4.0"
+    },
+    {
+      "name": "NVIDIA_CONTAINER_TOOLKIT_VERSION",
+      "value": "1.5.0"
+    },
+    {
+      "name": "NVIDIA_CONTAINER_RUNTIME_VERSION",
+      "value": "3.5.0"
+    }
+  ],
+  "build_operand_extra_docker_strategy": {
+    "from": {
+      "kind": "DockerImage",
+      "name": "nvcr.io/nvidia/cuda:11.2.1-base-ubi8"
```

Comment:
This avoid [this](https://gitlab.com/nvidia/container-toolkit/container-config/blob/a79c4dcf0c2e2d72f7b3578bc95f206fb05a0ddb/docker/Dockerfile.ubi8#L36) Docker hub pull - but we can only affect the last FROM in a multi-stage build. The [first one](https://gitlab.com/nvidia/container-toolkit/container-config/blob/a79c4dcf0c2e2d72f7b3578bc95f206fb05a0ddb/docker/Dockerfile.ubi8#L15) will still happen from Docker Hub, and we might hit our rate limit if we don't fix that.

Maybe we should ask nvidia to stop using Docker hub upstream, or [fork the repos and fix that ourselves](https://github.com/openshift-psap/ci-artifacts/pull/172/files#r637164737)

Reply:
> Maybe we should ask nvidia to stop using Docker hub upstream, or fork the repos and fix that ourselves

yes, this needs to be discssed with NVIDIA, pulling from DockerHub gets me build failure on Quay.io as well ...